project_national/housing_characteristics/Orientation.tsv

```diff
@@ -1074,3 +1074,4 @@ WY_Cheyenne.Muni.AP.725640	Multi-Family with 5+ Units	0.165906	0.165906	0.072205
 WY_Green.River-Greater.Green.River.Intergalactic.Spaceport.725744	Multi-Family with 5+ Units	0.165906	0.165906	0.072205	0.072205	0.184840	0.184840	0.077049	0.077049
 WY_Lander-Hunt.Field.725760	Multi-Family with 5+ Units	0.165906	0.165906	0.072205	0.072205	0.184840	0.184840	0.077049	0.077049
 WY_Sheridan.County.AP.726660	Multi-Family with 5+ Units	0.165906	0.165906	0.072205	0.072205	0.184840	0.184840	0.077049	0.077049
+"# Source: Microsoft Footprint Database, accessed 2018"									
```

Comment:
I mistakenly told you that we used "Microsoft Footprints Database". We have not used that yet. It should read "OpenStreetMap data queried by Radiant Labs"